### PR TITLE
Support extra fields for COSV

### PR DESCRIPTION
**Pull request by nulls**

### CODE REVIEW

File: src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt [hunk 1]
```diff
@@ -600,7 +754,34 @@ data class Severity(
...
+    )
+    @JsonProperty(
+        value = "score_num",
+        namespace = "",
+        required = false,
+        index = -1,
+        defaultValue = "",
+        access = JsonPropertyAccess.AUTO
+    )
+    val scoreNum: Float,
```
Comment (nulls) [inline src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt:784]: it's a number by description, but String by `format`.

**Comment by akuleshov7** [comment]
And readme?

**Review by akuleshov7 — APPROVED** [review]

**Comment by nulls** [comment]
> And readme?

updated + change group id for this repo to `com.saveourtool.cosv4k`

File: src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt [hunk 2]
```diff
@@ -770,3 +963,154 @@ enum class ReferenceType {
...
+    val contributions: String? = null,
+)
+
+/**
+ * the confirmation type of this vulnerability record
+ */
+enum class ConfirmType {
+    manual_confirmed,
+    algorithm_confirmed,
+    double_confirmed,
```
Comment (nulls) [inline src/commonMain/kotlin/com/saveourtool/osv4k/OsvSchema.kt:1140]: I'm to lazy, so decided to use not valid enum names, probably better to add `SerialName`, but will fix later

**Pull request merged**